### ISSUE TITLE
NTR: Vanillactic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8619]
New term request Vanillactic acid high in urine (segal_030513195810):   UMLS: 1969162

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6001002
- label: Elevated urinary vanillactic acid level
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: 
- def: The amount of vanillactic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8619
- synonyms:
